README.md

```diff
@@ -32,9 +32,42 @@ descopeClient := client.New()
 descopeClient := client.NewWithConfig(&client.Config{ProjectID: projectID})
 ```
 
-## Usage
+## Authentication Functions
 
-Here are some examples how to manage and authenticate users:
+These sections show how to use the SDK to perform various authentication/authorization functions:
+
+1. [OTP Authentication](#otp-authentication)
+2. [Magic Link](#magic-link)
+3. [Enchanted Link](#enchanted-link)
+4. [OAuth](#oauth)
+5. [SSO/SAML](#ssosaml)
+6. [TOTP Authentication](#totp-authentication)
+7. [Passwords](#passwords)
+8. [Session Validation](#session-validation)
+9. [Roles & Permission Validation](#roles--permission-validation)
+10. [Logging Out](#logging-out)
+
+## API Managment Function
+
+These sections show how to use the SDK to perform API management functions:
+
+1. [Setup](#setup-1)
```

Comment:
I see you include setup in the mgmt but not in the authentication, by purpose?

Reply:
I made it a little more clear in the table of contents